client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -1403,7 +1431,10 @@ const SubmitButtonHeaderWrapper = styled.div`
 	}
 `;
 
-const WPCheckoutWrapper = styled.div`
+const WPCheckoutWrapper = styled.div< {
+	isLargeViewport: boolean;
+	isStreamlinedPrice: boolean;
```

Comment:
`isStreamlinedPrice` might need to be optional: there another usage of  WPCheckoutWrapper that is without this prop and is causing typecheck fails. 
This could explain the failing Unit tests at the`Task type_check_client` task.

Reply:
Fixed!